README.md

```diff
@@ -4,13 +4,22 @@
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
+# About the Project
+**UGTeach** empowers undergraduate private tutors to efficiently manage payments, track student performance, and organize schedules. It streamlines tutoring operations, ensuring you stay organized and focused on helping your students achieve their academic goals.<br>
+
+  **Example usages:**
+  * As a private tutor, you can add student's personal information and their tuition details
+  * As a private tutor, you can view the list of students you are currently tutoring
+  * As a private tutor, you can view the amount of money you have earned from tutoring
+
+# More Information
+  * It is **written in OOP fashion**. 
+  * It is **written in Java**, and has about 6 kLoC.
   * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
+  * It is developed by a team of 5 from NUS Computer Science as a project for a module on software engineering (CS2103T). You can find more [about us here](https://ay2425s1-cs2103t-f14a-1.github.io/tp/AboutUs.html).
+
+* For the detailed documentation of this project, see the **[UG Teach Product Website](https://ay2425s1-cs2103t-f14a-1.github.io/tp/)**.
+
+
+# Acknowledgement
 * This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org/#contributing-to-se-edu) for more info.
```

Comment:
Acknowledge the original source of the code e.g.,
This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).